### PR TITLE
apply autoformatting; update to latest develop for dev/ufs-weather-model branch; fix wave_elevation export for coupling with CICE6

### PR DESCRIPTION
**Pull request by DeniseWorthen**
# Pull Request Summary
Apply autoformatting and white space removal to dev/ufs-weather-model branch. Update to latest develop. Fix export of wave_elevation spectrum to be updated at coupling frequency when coupled with CICE6.

## Description
Apply autoformatting and white space removal to dev/ufs-weather-model branch. Update to latest develop. No change to baselines for UWM regression tests (https://github.com/ufs-community/ufs-weather-model/pull/1474)

### Issue(s) addressed

fixes issues #796 and #797 for ufs-weather-model branch
fixes issue #843 
 
### Commit Message

Apply autoformatting and white space removal to dev/ufs-weather-model branch. Update to latest develop. Fix export of wave_elevation spectrum to be updated at coupling frequency when coupled with CICE6.

co-author: @mvertens

### Check list  

- [ ] Branch is up to date with the authoritative repository (NOAA-EMC) develop branch. 
- [ ] Checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop). 
- [ ] If a version number update is required, checked the [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number) checklist. 
- [ ] If a new feature was added, a regression test for testing the new feature is added. 
 
### Testing

* How were these changes tested?
* Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)
* Have the matrix regression tests been run (if yes, please note HPC and compiler)?
* Please indicate the expected changes in the regression test output, (Note the [list](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-develop#4-look-at-results) of known non-identical tests.)
* Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):

### CODE REVIEW

File: model/src/w3sdb1md.F90 [hunk 1]
```diff
@@ -1,349 +1,343 @@
...
+    !/
+    INTEGER                 :: IS
 #ifdef W3_S
-      INTEGER, SAVE           :: IENT = 0
+    INTEGER, SAVE           :: IENT = 0
 #endif
+    REAL*8                    :: HM, BB, ARG, Q0, QB, B, CBJ, HRMS, EB(NK)
+    REAL*8                    :: AUX, CBJ2, RATIO, S0, S1, THR, BR1, BR2, FAK
+    REAL                      :: ETOT, FMEAN2
 #ifdef W3_T0
```
Comment (MatthewMasarik-NOAA) [inline model/src/w3sdb1md.F90:177]: @DeniseWorthen is this block correct?

Comment (MatthewMasarik-NOAA) [reply model/src/w3sdb1md.F90:177]: @DeniseWorthen, whoops, I see `IK` and `ITH` are declared above.

**Review by MatthewMasarik-NOAA — COMMENTED** [review]
Visual inspection looks good to me.

**Comment by DeniseWorthen** [comment]
Update of WW3 in UWM has been rolled into UWM [https://github.com/ufs-community/ufs-weather-model/pull/1456](https://github.com/ufs-community/ufs-weather-model/pull/1456)

**Comment by BrianCurtis-NOAA** [comment]
We're ready for reviews/merge on the UFSWM side.

**Comment by MatthewMasarik-NOAA** [comment]
> We're ready for reviews/merge on the UFSWM side.

Great.  We'll merge now on this side.

**Review by MatthewMasarik-NOAA — APPROVED** [review]
All looks well.

**Comment by MatthewMasarik-NOAA** [comment]
Thanks @DeniseWorthen, for this huge step in cleaning up the WW3 code base!

**Pull request merged**